.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
     - EMBER_TRY_SCENARIO=ember-release
     - EMBER_TRY_SCENARIO=ember-beta
     - EMBER_TRY_SCENARIO=ember-canary
-    - EMBER_TRY_SCENARIO=ember-1.13.0
+    - EMBER_TRY_SCENARIO=ember-1.13.3
```

Comment:
I've noticed in the past and ran into it again, recompute and completing a rerender in time for tests.  This seems to have been resolved in 1.13.3